### ISSUE TITLE
"Undefined index: width" when trying to clone Super Table field

### ISSUE THREAD

**Issue by dommmel** [engram-design/FieldManager#31]
Thanks for your plugins! They have become indispensable for me and my clients. I am experiencing the following error when trying to clone a super tabel field:

```
Undefined index: width (/var/www/craft/plugins/supertable/fieldtypes/SuperTableFieldType.php:83)

#0 /var/www/craft/app/etc/errors/ErrorHandler.php(184): Craft\ErrorHandler->handleError()
#1 /var/www/craft/app/framework/base/CErrorHandler.php(133): Craft\ErrorHandler->handleError()
#2 /var/www/craft/app/framework/base/CApplication.php(834): Craft\ErrorHandler->handle()
#3 /var/www/craft/app/etc/web/WebApp.php(685): Craft\WebApp->handleError()
#4 /var/www/craft/plugins/supertable/fieldtypes/SuperTableFieldType.php(83): Craft\WebApp->handleError()
#5 /var/www/craft/app/services/FieldsService.php(502): Craft\SuperTableFieldType->prepSettings()
#6 /var/www/craft/app/services/FieldsService.php(535): Craft\FieldsService->validateField()
#7 /var/www/craft/plugins/fieldmanager/services/FieldManagerService.php(29): Craft\FieldsService->saveField()
#8 /var/www/craft/plugins/fieldmanager/controllers/FieldManagerController.php(94): Craft\FieldManagerService->saveField()
#9 /var/www/craft/app/framework/web/actions/CInlineAction.php(49): Craft\FieldManagerController->actionCloneField()
#10 /var/www/craft/app/framework/web/CController.php(308): CInlineAction->runWithParams()
#11 /var/www/craft/app/framework/web/CController.php(286): Craft\FieldManagerController->runAction()
#12 /var/www/craft/app/framework/web/CController.php(265): Craft\FieldManagerController->runActionWithFilters()
#13 /var/www/craft/app/framework/web/CWebApplication.php(282): Craft\FieldManagerController->run()
#14 /var/www/craft/app/etc/web/WebApp.php(815): Craft\WebApp->runController()
#15 /var/www/craft/app/etc/web/WebApp.php(285): Craft\WebApp->_processActionRequest()
#16 /var/www/craft/app/framework/base/CApplication.php(185): Craft\WebApp->processRequest()
#17 /var/www/craft/app/index.php(62): Craft\WebApp->run()
#18 /var/www/public/index.php(19): require_once()
```

Hope this is helpful. Cheers!

**Comment by engram-design** [comment]
Are you able to let me know what version of Super Table you're running? Does potentially look more like a Super Table issue...

**Comment by dommmel** [comment]
I was running the latest super table release at that time.
I _think_ the table I was tying to clone contained one dropdown menu and then only numbers. Thanks!

**Comment by engram-design** [comment]
This should be fixed with the latest Super Table release - [0.4.8](https://github.com/engram-design/SuperTable/releases/tag/0.4.8), but you might also like to update to Field Manager [1.5.1](https://github.com/engram-design/FieldManager/releases/tag/1.5.1) which fixes some bugs with Matrix / Super Table.

Let me know if you have any issues!
